pom.xml

```
@@ -84,6 +85,11 @@
       <artifactId>lombok</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.postgresql</groupId>
+      <artifactId>postgresql</artifactId>
+      <version>${postgresql.version}</version>
```

```suggestion
      <scope>runtime</scope>
```
The version of postgresql should be in sync with the spring libs that working with it. So I propose to use the dependency that `spring-boot-starter-parent` provides.
Also, we can use `runtime` scope for PostgreSQL driver.